pkgs/development/python-modules/frida-python/default.nix

```
@@ -32,8 +43,17 @@ buildPythonPackage rec {
     popd
   '';
 
+  NIX_LDFLAGS = toString (lib.optionals stdenv.hostPlatform.isDarwin [
+    "-framework AppKit"
+  ]);
+
   propagatedBuildInputs = [ typing-extensions ];
 
+  buildInputs = [ ]
+  ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.AppKit
+  ];
+
```

This is not really needed:

```suggestion
  buildInputs = lib.optionals stdenv.isDarwin [
    darwin.apple_sdk.frameworks.AppKit
  ];

```